fabric-chaincode-shim/src/main/java/org/hyperledger/fabric/ledger/Collection.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2019 IBM All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.fabric.ledger;
+
+/** Place holder. */
+public interface Collection {
+
+    /**
+     * Constant that can be used to refer to the 'Worldstate' collection explicitly.
+     */
+    String WORLD = "worldstate";
+
+    /** Gets the orignaizational collection name based on the mspid
+     * 
+     * 
+     * @param mspid String the MSP ID to map to a colleciton name
+     * @return name of the organizations collection
+     */
+    static String organizationCollection(String mspid){
```

Comment:
Was this meant to be in the first story? https://jira.hyperledger.org/browse/FABCJ-260